libs/community/langchain_community/document_loaders/chromium.py

```diff
@@ -63,6 +62,33 @@ async def ascrape_playwright(self, url: str) -> str:
             await browser.close()
         return results
 
+    def scrape_playwright(self, url: str) -> str:
```

Comment:
@hinthornw could you document behavior if the url is: 

1) prefixed with `file:///`
2) What happens on redirects?